zabbix-mysql-dump

```diff
@@ -214,7 +214,7 @@ elementIn () {
 #
 # CHECKS
 #
-if [ ! -x /usr/bin/mysqldump ]; then
+if [ ! -x `which mysqldump` ]; then
```

Comment:
Yes, although I usually prefer short syntax in this case I line the beauty if $().
Btw. thank you very much for your report and the PR! As soon as I find some time I'll merge and test all open PRs and make a new release.